server/app/views/v1/containers/_container.json.jbuilder

```diff
@@ -41,3 +41,7 @@ else
   json.overlay_cidr nil
 end
 json.network_settings container.network_settings
+if container.health_status
+  json.health_status container.health_status
+  json.health_status_at container.health_status_at
```

Comment:
Ok, so then we should return health json with a future proof format (as an object)?